src/doc/book/documentation.md

```diff
@@ -73,7 +73,7 @@ hello.rs:4 }
 ```
 
 This [unfortunate error](https://github.com/rust-lang/rust/issues/22547) is
-correct: documentation comments apply to the thing after them, and there's
+correct; documentation comments apply to the thing after them, and there's
```

Comment:
Yes. I read it as being independant but also part of/related to the following sentence though;
- This unfortunate error is correct. Documentation comments apply...
Which I then add a `;` to;
- This unfortunate error is correct; documentation comments apply...

This is the way I was taught to use it in University.
[Reference link: Semicolon](http://www.bristol.ac.uk/arts/exercises/grammar/grammar_tutorial/page_05.htm)
[Reference link: Colon](http://www.bristol.ac.uk/arts/exercises/grammar/grammar_tutorial/page_04.htm)